src/utils/dif.rs

```diff
@@ -21,6 +21,7 @@ pub enum DifType {
     Pdb,
     PortablePdb,
     Wasm,
+    Jvm,
```

Comment:
Do we really want this here? The file we create *is* just a `SourceBundle`, and no debug file on its own exists for JVM.

Reply:
In an internal discussion we didn't decide whether we actually need the new type but it was suggested to err on the side of defining a new one. Do you want me to reopen that conversation internally? Handling of `jvm` bundles is different from others in that files are renamed while bundling and lookup constructs the file path from `module` and `abs_path` if available.